storage/src/main/java/com/tencent/rss/storage/handler/impl/ComposedClientReadHandler.java

```diff
@@ -47,6 +56,25 @@ public ComposedClientReadHandler(ClientReadHandler... handlers) {
     if (size > 2) {
       this.coldDataReadHandler = handlers[2];
     }
+    if (size > 3) {
+      this.frozenDataReadHandler = handlers[3];
+    }
+  }
+
+  public ComposedClientReadHandler(Callable<ClientReadHandler>... creators) {
```

Comment:
Why we need this interface ？

Reply:
lazy create handler, we can reduce `list operation`.